### PR TITLE
refactor: setup unit group summaries with required fields, rename objs

### PR DESCRIPTION
**Pull request by emilyjablonski**
## Issue

- Closes #issue
- Addresses #issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

`unitsSummary` --> `unitGroups`
`unitsSummarized` --> `unitSummaries`
`unitsSummarized.byUnitType` --> `unitTypeSummary`
`unitsSummarized.byUnitTypeAndRent` --> `unitGroupSummary`
`unitsSummarized.hmi` --> `housholdMaxIncomeSummary`

Unit type on the summaries made an array
Added amiPercentageRange onto unitGroup summary
Added two new types for the unitTypeSummary and unitGroupSummary that reflects just the data they need

Removed all old logic from units-transformations, put in empty fxns for all three summaries.

Does no frontend uptake, frontend still references old fields (as was the case with the base PR), but the frontend will not break - it will just render empty tables. The following PR will build the summaries and do frontend uptake.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components

### CODE REVIEW

File: backend/core/src/migration/1645640266273-add-units-summary-ami-levels-entity.ts [hunk 1]
```diff
@@ -1,52 +1,88 @@
...
-        await queryRunner.query(`ALTER TABLE "units_summary" ADD "monthly_rent_as_percent_of_income" numeric(8,2)`);
-        await queryRunner.query(`DROP INDEX "IDX_b905b8bda3171b06c7a5d4d671"`);
-        await queryRunner.query(`DROP INDEX "IDX_1951c380e8091486b980008886"`);
-        await queryRunner.query(`DROP TABLE "units_summary_unit_type_unit_types"`);
-        await queryRunner.query(`DROP TABLE "units_summary_ami_levels"`);
-        await queryRunner.query(`DROP TYPE "units_summary_ami_levels_monthly_rent_determination_type_enum"`);
-        await queryRunner.query(`ALTER TABLE "units_summary" ADD CONSTRAINT "FK_0eae6ec11a6109496d80d9a88f9" FOREIGN KEY ("unit_type_id") REFERENCES "unit_types"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`);
-    }
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE units_summary RENAME TO unit_group`)
```
Comment (emilyjablonski) [inline backend/core/src/migration/1645640266273-add-units-summary-ami-levels-entity.ts:7]: Only change on up and down

Comment (emilyjablonski) [inline backend/core/src/migration/1645640266273-add-units-summary-ami-levels-entity.ts:7]: Only change in up/down

**Review by seanmalbert — APPROVED** [review]
@emilyjablonski ,
Can you update this to keep the units-transformations working for individual units (even though we won't be using them in Detroit, it'll make keeping Detroit in sync easier) and create a unit-group-transformations with the full changes of units-transformations. Once that's done, I think you're good to merge.

**Comment by seanmalbert** [comment]
@pbn4 , fyi, @emilyjablonski created this to make working with unit groups/summaries less confusing.

**Comment by emilyjablonski** [comment]
I think the migration looks okay, I'm going to merge into the other branch

**Pull request merged**